windowsforms/Sudoku/VB/readme.md

```diff
@@ -35,9 +35,9 @@ description: "Sudoku is a number puzzle game, played on a nine by nine cell grid
 
 ## Description
 
-![Sudoku image](Sudoku-VB.jpg)
+This Windows Forms sample was ported to .NET Core from a [previous .NET Framework 4.5 sample](https://github.com/microsoftarchive/msdn-code-gallery-community-s-z/tree/master/Sudoku).
```

Comment:
Do you want to say .NET Core 3.0, or 3.1?

Reply:
```suggestion
This Windows Forms sample was ported to .NET Core 3.1 from a [previous .NET Framework 4.5 sample](https://github.com/microsoftarchive/msdn-code-gallery-community-s-z/tree/master/Sudoku).
```